### DISCUSSION TITLE
The terminal process failed to launch

### DISCUSSION THREAD

**Discussion by pauljurczak** [pycom/pymakr-vsc#256] [Q&A]
This is most likely a newbie error. I'm using VS Code:
```
Version: 1.68.0
Commit: 4af164ea3a06f701fe3e89a2bcbb421d2026b68f
Date: 2022-06-08T11:49:57.055Z
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Linux x64 5.13.0-44-generic
```
with Pymakr - Preview v2.16.4. I'm getting `The terminal process failed to launch: Path to shell executable "node" does not exist` error when trying to create terminal.

![pymakr-1](https://user-images.githubusercontent.com/4933032/173109943-e7fd0587-4af3-4eed-86d0-31a9f5008440.png)

The LED blink example I'm trying works fine with Thonny. VS Code creates terminal just fine when working with Platform IO or desktop Python and C++. What am I missing?

**Comment by jakobrosenberg** [comment]
It's my impression that VSCode bundles with [node](https://nodejs.org/en/), but I could be wrong on this. At any rate, your local installation can't find node. First make sure that your installation followed the instructions [here](https://code.visualstudio.com/) as various market places sometimes mess up the VSCode install. Something we've seen with bindings in the past.

If that doesn't work, I'll see if I can build binaries for Linux. In the meantime, you can solve the issue by installing Node, but I understand if this is not desired.

**Reply by MrclM** [reply]
Hello, I had the same problem.

```
Version: 1.68.0
Commit: 4af164ea3a06f701fe3e89a2bcbb421d2026b68f
Date: 2022-06-08T11:49:57.055Z
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Linux x64 5.15.0-37-generic snap
```

Installing nodejs from the dpkg mananger results in another problem:
![Screenshot from 2022-06-11 23-11-07](https://user-images.githubusercontent.com/107324281/173205095-3da02fb1-9a31-4546-b591-3e25365246f8.png)

What could be the reason for this issue? Thanks in advance!

**Comment by pauljurczak** [comment]
I decided to give MicroPython a break for a while. Going back to C/C++ for embedded development.
